### DISCUSSION TITLE
I found a mime detector library

### DISCUSSION THREAD

**Discussion by sandreas** [Zeugma440/atldotnet#157] [Ideas]
Hey,

I saw `atldotnet` using mimetypes in `Track` for sume purposes... Today I found a library, that could be useful:

https://github.com/MediatedCommunications/Mime-Detective

What do you think?

**Comment by Zeugma440** [comment]
Thanks. That's good to know, but I'm pretty satisfied with what I've done on my end. I've starred that one for future projects 😉

**Comment by sandreas** [comment]
> Thanks. That's good to know, but I'm pretty satisfied with what I've done on my end. I've starred that one for future projects 😉

Yeah I also found out, that there is something fishy with the LICENSE going on... see https://github.com/MediatedCommunications/Mime-Detective/issues/17

That was not obvious to me...
